src/Templates/src/templates/maui-mobile/Platforms/Android/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
-	<application android:allowBackup="true" android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round" android:supportsRtl="true"></application>
+	<application android:allowBackup="true" android:supportsRtl="true"></application>
```

Comment:
@davidortinau @Redth  I have updated the android templates to no longer have the icon things because it is now automatically "injected" at build time. However, as with all things, they can be overridden by simply adding them back and Resizetizer will do nothing.